.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -4,23 +4,15 @@ labels: 'needs review'
 body:
   - type: markdown
     attributes:
-      value: Thanks for taking the time to file a bug report! Please fill out this form as completely as possible.
+      value: Thanks for taking the time to file a bug report! If you are convinced that you have found a bug in Expo tools, this is the right place. Please fill out this entire form.
   - type: markdown
     attributes:
-      value: If you leave out sections there is a high likelihood your issue will be closed. If you have a question, not a bug report, please post it on our [forums](https://forums.expo.dev/) instead.
+      value: If you have a question or you think your issue might be caused by your application code, you can get help from the community on the [forums](https://forums.expo.dev/) or on [Discord](https://chat.expo.dev).
   - type: textarea
     attributes:
       label: Summary
-      description: Describe the issue in 1 or 2 sentences
-      placeholder: Clearly describe what the expected behavior is vs. what is actually happening. This should be as short as possible, while still communicating all the necessary information. If your summary is just 'X library/method isn't working', then you need to continue debugging yourself and provide more information.
-    validations:
-      required: true
-  - type: dropdown
-    attributes:
-      label: Managed or bare workflow? If you have `ios/` or `android/` directories in your project, the answer is bare!
-      options:
-        - managed
-        - bare
+      description: Describe the issue
+      placeholder: Clearly describe what the expected behavior is vs. what is actually happening. Try to be concise and precise. If all you have to report is that 'X library/method isn't working', then will need to [continue debugging on your own](https://expo.fyi/manual-debugging) to more precisely articulate your issue before proceeding.
```

Comment:
Maybe "define" as an alternative to "articulate"? The latter can be evocative of fluency in language, technical jargon, etc., which may be absent in an otherwise good issue description.

Reply:
Alt:

Clearly describe what the expected behavior is and what instead is actually happening. Be concise and precise in your description. Ex: if you report that 'X library/method isn't working', then you will need to [continue debugging on your own](https://expo.fyi/manual-debugging) to more precisely define your issue before proceeding.